controllers/object_controls.go

```diff
@@ -1403,6 +1403,25 @@ func transformForRuntime(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec,
 		obj.Spec.Template.Spec.Volumes = append(obj.Spec.Template.Spec.Volumes, configVol)
 	}
 
+	// Handle any additional runtime config sources
+	const runtimeConfigSourceFile = "file"
+	if runtimeConfigSources := getContainerEnv(container, "RUNTIME_CONFIG_SOURCE"); runtimeConfigSources != "" {
+		var sources []string
+		for _, runtimeConfigSource := range strings.Split(runtimeConfigSources, ",") {
+			parts := strings.SplitN(runtimeConfigSource, "=", 2)
+			if len(parts) == 1 || parts[0] != runtimeConfigSourceFile {
+				sources = append(sources, runtimeConfigSource)
+				continue
+			}
+			// We transform the host path to a container path by prepending "/host" to the file
+			// path. This works because the toolkit container has the host's root filesystem
+			// mounted as read-only at "/host"
+			sourceConfigFile := "/host" + parts[1]
```

Comment:
One nit is that this ASSUMES that `parts[1]` is an absolute path.

Reply:
I have updated to using `filepath.Join()` to be more robust. Although note, I would only expect the toolkit to be able to find the source config if `parts[1]` is an absolute path.